docs/DeveloperGuide.md

```diff
@@ -77,6 +77,7 @@ Manage contacts and schedule of students faster than a typical mouse/GUI driven
 
       Use case resumes at step 2.
 
+[](#use-case-update-a-student-contact-detail)
```

Comment:
Seems to be an extra line here?

Reply:
this is a hyperlink so that the user can click the `update` word in line 158 can automatically jump to here for better read experience.